docs/Data_Portal/Users_Guide/Developers_Guide.md

```diff
@@ -461,64 +506,69 @@ for altering the current cohort:
 * `clearCohortFilters`
 
 Note that all of these operations are applied to the current cohort. The current cohort is the cohort that is currently
-being displayed in the Cohort Management Bar. The current cohort can be accessed via the `selectCurrentCohort` selector.
+being displayed in the Cohort Management Bar. Programmatically current cohort can be accessed via the `selectCurrentCohort` selector.
```

Comment:
Maybe... :
"The current cohort can be programmatically accessed..."? (I thought it was ok before the change too)